zwave_js/CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 # Changelog
 
+## 0.5.0
+
+### Features
+
+- Z-Wave JS: Map more Basic CC values to more useful command classes
+- Z-Wave JS: Add Z-Wave long range support
+- Z-Wave JS Server: Add Z-Wave long range support
+- Addon: Add support for collecting Z-Wave Long Range security keys
+
+### Bug fixes

```

Comment:
- Z-Wave JS: Fixed a crash that could happen in some cases during the Configuration CC interview

only visible in Z-Wave JS UI, but so are the NVM and association related fixes:

- Z-Wave JS: Fixed an issue where provisioning entries could disappear